domain-models-runtime/src/main/java/org/folio/rest/persist/PostgresClient.java

```diff
@@ -885,50 +885,55 @@ public void saveBatch(String table, JsonArray entities, Handler<AsyncResult<Resu
   }
 
   public void saveBatch(String table, JsonArray entities, String column, Handler<AsyncResult<ResultSet>> replyHandler) {
+    client.getConnection(res -> {
+      if (res.failed()) {
+        replyHandler.handle(Future.failedFuture(res.cause()));
+        return;
+      }
+      saveBatch(res, table, entities, column, x -> {
+        res.result().close();
+        replyHandler.handle(x);
+      });
+    });
+  }
+
+  public void saveBatch(AsyncResult<SQLConnection> sqlConnection, String table, JsonArray entities, String column, Handler<AsyncResult<ResultSet>> replyHandler) {
     long start = System.nanoTime();
     if (entities == null || entities.isEmpty()) {
       // return empty result
       ResultSet resultSet = new ResultSet(
-          Collections.singletonList(idField), Collections.emptyList(), null);
+        Collections.singletonList(idField), Collections.emptyList(), null);
       replyHandler.handle(Future.succeededFuture(resultSet));
       return;
     }
-    client.getConnection(res -> {
-      if (res.failed()) {
-        replyHandler.handle(Future.failedFuture(res.cause()));
+    if (log.isInfoEnabled()) {
+      log.info("starting: saveBatch size=" + entities.size());
+    }
+    StringBuilder sql = new StringBuilder()
+      .append(INSERT_CLAUSE)
+      .append(schemaName).append(DOT).append(table)
+      .append(" (").append(column).append(") VALUES (?)");
```

Comment:
` String column` - I assume that this parameter is redundant to save objects in batch

Reply:
Redundant in the "inner" function?? Are you saying that the inner function do not need a column OR that  you just don't need it for YOUR purpose.. Just like the previous one.. WOuld like a version that takes List<Object> and no column parameter?

Reply:
In our case, the `column `parameter is not needed but may be needed for the future. Currently, we are using in the `mod-calendar` a list of objects without a  `column` parameter